package.json

```diff
@@ -29,6 +29,7 @@
     "typescript-eslint": "^0.0.1-alpha.0"
   },
   "dependencies": {
+    "@babel/traverse": "^7.23.2",
```

Comment:
Why this dependency `@babel/traverse` has been added here? Seems not been used anywhere?

/cc @CluEleSsUK

Reply:
Great question - I have no idea, it seems unused; I will remove it